static/src/stylesheets/components/pasteup-buttons/src/_buttons.scss

```diff
@@ -101,105 +101,117 @@ Show more button:
 .button--primary {
```

Comment:
:-1: this is bower component - you need to update that and pull this in. have i just missed that?